### ISSUE TITLE
[Suggestion] Ender Utilities

### ISSUE THREAD

**Issue by epiz** [CyanideX/Unity#123]
I would like to see support for [Ender Utilities](https://minecraft.curseforge.com/projects/ender-utilities).

**Comment by Mattabase** [comment]
You can make up for the lack of IC2Classic by doing this one @CyanideX =P

**Comment by ProfessorProspector** [comment]
tbh this one would probably be more fun to do anyways since he already did IC2 exp

**Comment by CyanideX** [comment]
Oh wow, those colors! 
![image](https://user-images.githubusercontent.com/3106700/28147473-99116466-6735-11e7-80ad-8e61a3d64598.png)
I'll see what I can do when I have some more free time.

**Comment by kragnoth** [comment]
Yes please, I love the concept but the textures are definitely programmer level. (still better than my textures though)
